Practical2/Customer_Account/Customer_Account/Program.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Customer_Account
+{
+    internal class Customer_Account
+    {
+        public string bankName;
+        public long customerAccountNo;
+        public string customerName;
+
+        public Customer_Account(long customer_accountNo, string customer_name)
+        {
+            customerAccountNo = customer_accountNo;
+            customerName = customer_name;
+        }
+        //printing the information of the customer - CustomerName,BankName,CustomerAccountNumber.
```

Comment:
Proper access modifiers are not used for class/method.
Use of this keyword to access current variable.